releases/nightly-build/compose-files/docker-compose-nexus-mongo.yml

```diff
@@ -24,6 +24,10 @@ x-common-env-variables: &common-variables
   Clients_CoreData_Host: edgex-core-data
   Clients_Logging_Host: edgex-support-logging
   Logging_EnableRemote: "true"
+  Databases_Primary_Type: mongodb
+  Databases_Primary_Host: edgex-mongo
+  Databases_Primary_Port: 27017
+  edgex-support-logging_Writable_Persistence: database
```

Comment:
This one has different prefix from others withe extra `edgex-`

Reply:
Modified.